packages/loot-core/src/server/accounts/parse-file.test.ts

```diff
@@ -39,7 +39,8 @@ async function importFileWithRealTime(
   global.restoreFakeDateNow();
 
   if (transactions) {
-    transactions = transactions.map(trans => ({
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    transactions = (transactions as any[]).map(trans => ({
```

Comment:
What's the problem here?

Reply:
`transactions` is `unknown[]` for now (since the different file types return different shapes). Ideally we would move the logic for importing more fully into `loot-core` and then either directly return transaction objects or insert them into the database (see #1085 for motivation), but that’s out of scope for this PR.